tokio-util/Cargo.toml

```
@@ -21,15 +21,16 @@ categories = ["asynchronous"]
 default = []
 
 # Shorthand for enabling everything
-full = ["codec", "compat", "io-util", "time", "net", "rt"]
+full = ["codec", "compat", "io-util", "time", "net", "rt", "join-map"]
 
 net = ["tokio/net"]
 compat = ["futures-io",]
 codec = []
 time = ["tokio/time","slab"]
 io = []
 io-util = ["io", "tokio/rt", "tokio/io-util"]
-rt = ["tokio/rt", "tokio/sync", "futures-util", "hashbrown"]
+rt = ["tokio/rt", "tokio/sync", "futures-util"]
+join-map = ["rt", "hashbrown"]
```

Would this be sufficient?
```suggestion
join-map = ["tokio/rt", "hashbrown"]
```

---

Hmm. The `mod task` requires the `rt` feature, but ultimately yes it should work with just `tokio/rt`

---

We can always mark `mod task` with `#[cfg(any(rt, join_map))]`.

---

Fair enough